src/mcp/tools/core/use_project.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { setNewActive } from "../../../commands/use.js";
+import { mcpError, toContent } from "../../util.js";
+
+export const use_project = tool(
+  {
+    name: "use_project",
+    description: "Select a Firebase Project to use for subsequent tool calls.",
+    inputSchema: z.object({
+      project: z.string().describe("The project id, number, or alias to use."),
+    }),
+    annotations: {
+      title: "Select a Firebase Project to use for subsequent tool calls.",
+      readOnlyHint: false,
+    },
+    _meta: {
+      requiresAuth: true,
```

Comment:
Does it require `auth`?

`firebase use` can be run w/o logging in, right?

Reply:
We check that you have access to the project if it is a real one, so its probably simpler to requrie auth for now.

We can always revist this once we add emulator support.